client/sites-dashboard/components/sites-table-row.tsx

```diff
@@ -205,53 +258,77 @@ export default memo( function SitesTableRow( { site }: SiteTableRowProps ) {
 						</ListTileTitle>
 					}
 					subtitle={
-						<ListTileSubtitle>
-							<SiteUrl href={ siteUrl } title={ siteUrl }>
+						isDeleted ? (
+							<span>
 								<Truncated>{ displaySiteUrl( siteUrl ) }</Truncated>
-							</SiteUrl>
-						</ListTileSubtitle>
+							</span>
+						) : (
+							<ListTileSubtitle>
+								<SiteUrl href={ siteUrl } title={ siteUrl }>
+									<Truncated>{ displaySiteUrl( siteUrl ) }</Truncated>
+								</SiteUrl>
+							</ListTileSubtitle>
+						)
 					}
 				/>
 			</Column>
+			{ ! isDeleted && (
+				<Column tabletHidden>
+					<SitePlan site={ site } userId={ userId } />
+				</Column>
+			) }
 			<Column tabletHidden>
-				<SitePlan site={ site } userId={ userId } />
-			</Column>
-			<Column tabletHidden>
-				<WithAtomicTransfer site={ site }>
-					{ ( result ) =>
-						result.wasTransferring ? (
-							<TransferNoticeWrapper { ...result } />
-						) : (
-							<>
-								<div>
-									{ translatedStatus }
-									<SiteLaunchNag site={ site } />
-								</div>
-								{ isDIFMInProgress && (
-									<BadgeDIFM className="site__badge">{ __( 'Express Service' ) }</BadgeDIFM>
-								) }
-							</>
-						)
-					}
-				</WithAtomicTransfer>
+				{ isDeleted ? (
+					<span>{ __( 'Deleted' ) } </span>
+				) : (
+					<WithAtomicTransfer site={ site }>
+						{ ( result ) =>
+							result.wasTransferring ? (
+								<TransferNoticeWrapper { ...result } />
+							) : (
+								<>
+									<div>
+										{ translatedStatus }
+										<SiteLaunchNag site={ site } />
+									</div>
+									{ isDIFMInProgress && (
+										<BadgeDIFM className="site__badge">{ __( 'Express Service' ) }</BadgeDIFM>
+									) }
+								</>
+							)
+						}
+					</WithAtomicTransfer>
+				) }
 			</Column>
 			<Column tabletHidden>
 				{ site.options?.updated_at ? <TimeSince date={ site.options.updated_at } /> : '' }
 			</Column>
-			<StatsColumnStyled tabletHidden>
-				{ inView && (
-					<>
-						{ hasStatsLoadingError ? (
-							<StatsOffIndicator />
-						) : (
-							<a href={ `/stats/day/${ site.slug }` }>
-								<StatsSparkline siteId={ site.ID } showLoader={ true } />
-							</a>
-						) }
-					</>
-				) }
-			</StatsColumnStyled>
-			<Column style={ { width: '24px' } }>{ inView && <SitesEllipsisMenu site={ site } /> }</Column>
+			{ ! isDeleted && (
+				<StatsColumnStyled tabletHidden>
+					{ inView && (
+						<>
+							{ hasStatsLoadingError ? (
+								<StatsOffIndicator />
+							) : (
+								<a href={ `/stats/day/${ site.slug }` }>
+									<StatsSparkline siteId={ site.ID } showLoader={ true } />
+								</a>
+							) }
+						</>
+					) }
+				</StatsColumnStyled>
+			) }
+			{ isDeleted ? (
+				<Column style={ { textAlign: 'right' } }>
+					<Button scary busy={ isPending } disabled={ isPending } onClick={ handleRestoreSite }>
+						{ __( 'Restore site ' ) }
```

Comment:
Would be good if we could trim the translatable strings:

```suggestion
						{ __( 'Restore site' ) }
```